### PR TITLE
feat: add icon sticker gif and brand giphy

### PR DESCRIPTION
**Pull request by yorbi-dp**
## Description
- add new icons: sticker and gif
- add brand giphy

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).

## Obligatory GIF (super important!)
Screenshot 2023-12-20 at 5 38 54 PM

### CODE REVIEW

File: packages/dialtone-icons/src/keywords.json [hunk 1]
```diff
@@ -519,6 +520,12 @@
         "emoji",
         "activity"
       ],
+      "gif-fill": [
+        "giphy"
+      ],
```
Comment (francisrupert) [inline packages/dialtone-icons/src/keywords.json]: @yorbi-dp,

Marshall changed this to just `gif`, removing the `-fill`. Also he hadn't yet merged the Figma branch, even though I approved him to do so. Go ahead and merge it.

Meanwhile, I think there's an outstanding task to change all of the icons to `color/foreground/primary`. He might not have gotten to it, but that should definitely be its own branch.

File: packages/dialtone-icons/src/keywords.json [hunk 2]
```diff
@@ -2298,4 +2308,4 @@
       ]
     }
   }
-}
\ No newline at end of file
+}
```
Comment (juliodialpad) [inline packages/dialtone-icons/src/keywords.json]: Do you have a json linter enabled? I think this should have the blank line at the bottom, I had issues with this previously on releases.

Maybe your linter is removing that line or we have something missconfigured.

**Review by juliodialpad — APPROVED** [review]
From the engineering perspective this is ok for now, I'll check the issue related to the blank at the end of the file.

**Review by braddialpad — APPROVED** [review]
Good by me after the changes francis suggested.

File: packages/dialtone-icons/src/keywords.json (hunk 1 above)
Comment (yorbi-dp) [reply packages/dialtone-icons/src/keywords.json]: Thanks for the heads up! Fixed in the lastest commit.

Just merged the branch. Will do the change of colors in a new one!

File: packages/dialtone-icons/src/keywords.json (hunk 2 above)
Comment (yorbi-dp) [reply packages/dialtone-icons/src/keywords.json]: Oh, thanks! Not sure why it got deleted, I don't have any json linter 😖 
Fixed in the lastest commit.

**Pull request merged**